test/integration/cart_integration_test.rb

```diff
@@ -41,7 +41,26 @@ class CartIntegrationTest < ActionDispatch::IntegrationTest
       assert page.has_content?("coffee")
       assert page.has_content?("1")
     end
+  end
+
+  test "an authenticated user can delete an item from the cart" do
+    authenticated_user = create(:user)
+    ApplicationController.any_instance.stubs(:current_user).returns(authenticated_user)
+    project = create(:project)
+
+    visit "/"
+    click_link_or_button("Public Works1")
```

Comment:
Instead of putting Public Works1 here, maybe do project.categories.first.name instead so that our factories can be changed more easily